controllers/event_rsvp.rb

```diff
@@ -43,14 +43,20 @@ def remove
     end
   end
 
+  def get_users
+    users = User.find(rsvp_params[:user_ids])
+    serializer = ::ActiveModel::ArraySerializer.new(users, each_serializer: BasicUserSerializer)
```

Comment:
@fzngagan I know I said use the ActiveModel::ArraySerializer, but let's actually use the ``serialize_data`` method in the ApplicationController here (and remove the render statement). This makes it easier to manage:

1. The serialization of the array
2. The scope (i.e. the guardian); and
3. Serialize and render in a single line

Reply:
@angusmcleod You're probably talking about `render_serialized` method right?

Reply:
@fzngagan Yeah, that's the one

Reply:
@fzngagan The other thing we need to better account for here is security. At the moment it seems as if anyone can get the details of any user_ids they send to this endpoint!